config.yaml

```diff
@@ -1,7 +1,8 @@
 admin:
   # For GRPC endpoints you might want to use dns:///flyte.myexample.com
-  endpoint: dns:///flyte.myexample.com
-  authType: Pkce
+  endpoint: dns:///localhost:30081
+  insecure: true
+#  authType: Pkce
```

Comment:
Exclude this file change

Reply:
why, I am planing to use same config file for sandbox

Reply:
Ok . can you remove the commented out line then.